### PR TITLE
ddl: cleanup redundant globally settings for tests

### LINKED ISSUE #32673: Cleanup redundant globally settings for tests in ddl package

### PR DESCRIPTION
**Pull request by tisonkun**
### What problem does this PR solve?

Issue Number: close #32673

```release-note
None
```

### CODE REVIEW

File: ddl/column_change_test.go [hunk 1]
```diff
@@ -48,9 +47,6 @@ type testColumnChangeSuiteToVerify struct {
 }
 
 func TestColumnChangeSuite(t *testing.T) {
-	_, err := infosync.GlobalInfoSyncerInit(context.Background(), "t", func() uint64 { return 1 }, nil, true)
```
Comment (tisonkun) [inline ddl/column_change_test.go:51]: All merged into `TestMain`.

File: ddl/ddl_test.go [hunk 2]
```diff
@@ -311,7 +284,7 @@ func buildRebaseAutoIDJobJob(dbInfo *model.DBInfo, tblInfo *model.TableInfo, new
 	}
 }
 
-func (s *testDDLSuiteToVerify) TestGetIntervalFromPolicy() {
```
Comment (tisonkun) [inline ddl/ddl_test.go:314]: An empty suite, cleanup the usage.

**Review by hawkingrei — APPROVED** [review]

**Review by morgo — APPROVED** [review]

**Pull request merged**